vertx-db2-client/src/test/java/io/vertx/db2client/TableJoinTest.java

```diff
@@ -0,0 +1,100 @@
+package io.vertx.db2client;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.sqlclient.Row;
+
+/**
+ * Tests for table joins which are documented here:
+ * https://www.ibm.com/support/knowledgecenter/SSEPEK_10.0.0/intro/src/tpc/db2z_joindatafromtables.html
+ */
+@RunWith(VertxUnitRunner.class)
+public class TableJoinTest extends DB2TestBase {
+	
+	@Test
+	public void testColumnRename(TestContext ctx) {
+		connect(ctx.asyncAssertSuccess(conn -> {
+			conn.query("SELECT immutable.id AS \"IMM ID\"," +
+		                      "immutable.message AS IMM_MSG," +
+					          "Fortune.id AS FORT_ID," +
+		                      "Fortune.message AS \"FORT ID\" FROM immutable " + 
```

Comment:
This is a nit and completely irrelevant to the test itself, but on line 25, I would suggest changing \"FORT ID\" to \"FORT MSG\" to remove confusion with the line above.

Reply:
good catch, I will correct that in a follow-on PR